### DISCUSSION TITLE
Merit certificate not working

### DISCUSSION THREAD

**Discussion by saiakhil46** [Sunbird-Lern/Community#16] [Issues]
We are not able to get merit certificates to work, completion certificates are working. Steps taken

Create a merit certificate i.e. specify a passing percent when adding a certificate to a batch
Complete the course as a learner and gain more marks than the passing
Expect merit certificate to get generated

@gandham-santhosh @komalm @Renuka-S

**Comment by saiakhil46** [comment]
As suggested i have moved https://github.com/project-sunbird/sunbird-community/discussions/805 to here

@pritha-tarento As you said i have checked the connectivity from pods to redis and it is fine, can you help on further

**Reply by pc0202** [reply]
@saiakhil46 please share the current failed logs

**Reply by saiakhil46** [reply]
@pritha-tarento Now the flink jobs are stable after updating correct port in job configmap. 
I asked the @komalm and team to check and confirm.
